src/core/QueryManager.ts

```diff
@@ -361,6 +367,28 @@ export class QueryManager<TStore> {
         query: mutation.document,
         variables: mutation.variables,
       });
+      if (isExecutionPatchResult(result) && isNonEmptyArray(result.incremental)) {
```

Comment:
```suggestion
      if (isExecutionPatchIncrementalResult(result) && isNonEmptyArray(result.incremental)) {
```

Would it make sense to use `isExecutionPatchIncrementalResult` here? It might be a bit more meaningful since this chunk of data operates on incremental chunks

Reply:
Ah that makes sense. I wasn't seeing that connection when reviewing. Your solution works great. Thanks!